development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -149,7 +149,7 @@ manager.
 
 .. note:: If you are compiling Godot for production use, then you can
           make the final executable smaller and faster by adding the
-          SCons option ``target=release_debug``.
+          SCons option ``target=template_release``.
```

Comment:
Shouldn't we also use `production=yes` here?

Reply:
added it here